odata-protocol/1 Introduction.md

```diff
@@ -52,6 +52,9 @@ Setting a complex property to a different type|
 Control information to prevent updates| 
 [2021](https://github.com/oasis-tcs/odata-specs/issues/2021)
 [Section ##ReplaceaCollectionofEntities]| Semantics of `continue-on-error` when replacing a collection of entities | [358](https://github.com/oasis-tcs/odata-specs/issues/358)
+[Section ##InvokinganAction]| 
```

Comment:
Will there be a change to that section, or is that change already part of a different PR?

Reply:
Prior to this PR, the change about omitting collection-valued parameters was in a section in OData-JSON. Now it is in this section.